### ISSUE TITLE
Reduce how often Ack are stored

### ISSUE THREAD

**Issue by patriknw** [akka/akka#28721]
The StoreMessageConfirmed is already write behind, since we only guarantee at-least once.
Each Ack will store a StoreMessageConfirmed, but we could accumulate those and to reduce number of stored events.

**Comment by jewertow** [comment]
I could implement it, but I see two approaches:
1. accumulate specified number of `Ack` and then send single `StoreMessageConfirmed`
2. accumulate all `Ack` in specified period and then send `StoreMessageConfirmed` if any exist.
Additionally the specified number or period could also be configurable, do you think should it be?

**Comment by patriknw** [comment]
Yes, I think there should be a configuration to define that the confirmations are stored after a given number of Acks, or after a duration.

We could wait with the duration part until we know how #28720 evolves, because the periodic `Request` messages will also trigger `StoreMessageConfirmed`.

There is no need to accumulate all Acks. For StoreMessageConfirmed with the same confirmationQualifier we only need to keep the last (highest seqNr) and store that when flushing.
`confirmationQualifier` is used by work pulling and sharding to keep track of confirmed seqNr per destination (worker/entity). For point-to-point the confirmationQualifier is always the same (`""`).
